docs/Home/index.js

```diff
@@ -23,6 +35,25 @@ const Foo = () => (
 );
       `}
     </Code>
+    <h2>Contribute</h2>
+    <p>
+      Check out our{' '}
+      <Link href="https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md">
+        contribution guide!
+      </Link>
+    </p>
+    <p>
+      Found a bug?{' '}
+      <Link href="https://github.com/dequelabs/cauldron/issues">
+        Open an issue!
```

Comment:
Do we want to create a guide for opening issues?

Reply:
We should probably be using [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)